pkg/chains/formats/slsa/v1/intotoite6.go

```diff
@@ -14,15 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package intotoite6
+package v1
```

Comment:
Good call out. 
No, it's not. This was causing confusion for me as well in the beginning. 
See https://github.com/tektoncd/chains/issues/597#issuecomment-1355191134 for more context. 

Chains should have its own versioning scheme since which maps to the true slsa since Chains will also have iterations of the provenance which we improve over time. I guess, we can discuss this a bit more at the Chains WG today.

Reply:
Let's drop a README in the slsa folder to document this.

Reply:
Since you approved and lgtm-ed it, I will do this in the next PR :)